### ISSUE TITLE
hetzner cloud network settings

### ISSUE THREAD

**Issue by Happyfeet01** [mailcow/mailcow-dockerized#4007]
Hi,

aktuell soll man laut den Docs die Einstellungen ändern in der Hetzner Cloud. Ist aber nur bei Ubuntu 18.04 der Fall. https://mailcow.github.io/mailcow-dockerized-docs/prerequisite-system/

Bei Ubuntu 20.04 muss nichts geändert werden. 

Hier ist es /etc/netplan/50-cloud-init.yaml
@andryyy

**Comment by m4c3** [comment]
you create a pull request in https://github.com/mailcow/mailcow-dockerized-docs?
also this relates to https://github.com/mailcow/mailcow-dockerized-docs/issues/219

**Comment by MAGICCC** [comment]
Indeed, we can add a infobox that it's only affecting 18.04 until it's EoL

**Comment by strarsis** [comment]
Would this [50-cloud-init.yml](https://gist.github.com/strarsis/970f77b766921841b7ae453c74e970b1) for Hetzner help?

**Comment by chrislp** [comment]
I am not sure if this is the right  place. But for Hetzner, the outgoing mail ports are blocked by default and cannot unblocked without help from their support. I might make sense to add it to the docs. See here:

<img width="944" alt="Image" src="https://github.com/user-attachments/assets/2a5a05a0-f359-4379-a187-7af6341daa40" />

**Comment by strarsis** [comment]
The Hetzner Cloud [firewall](https://docs.hetzner.com/cloud/firewalls/getting-started/creating-a-firewall/) can be configured.

**Comment by chrislp** [comment]
> The Hetzner Cloud [firewall](https://docs.hetzner.com/cloud/firewalls/getting-started/creating-a-firewall/) can be configured.

While this is correct, the ports 25 and 465 are not blocked in the firewall. This is what makes this note so important:   
In the firewall the ports are open, yet you cannot use them. You have to send a request to their support team.    
Also important: They will only unblock it after you are customer for 1 month or more and paid at least one invoice.

See also here:    
https://docs.hetzner.com/de/cloud/servers/faq#warum-kann-ich-keine-mails-von-meinem-server-verschicken     
https://www.reddit.com/r/hetzner/comments/lb2o13/does_hetzner_block_port_25/

**Comment by strarsis** [comment]
@chrislp: Good point!
